dist/pinny.js

```diff
@@ -443,6 +445,19 @@
             return $(document.activeElement);
         },
 
+        /*
+        * In iOS, there is an issue when soft keyboard triggers, position fixed
+        * elements on screen don't get updated, distorting the UI.
+        * Fix: blur keyboard on orientationchange
+        */
+        _blurOnOrientationChange: function($pinny) {
+            if ($.os.ios) {
+                $(window).on('orientationchange', function(e) {
+                    $pinny.find(':focus').blur();
```

Comment:
Are you sure this works with Zepto? I don't think these custom pseudos work unless you're using a Sizzle based selector engine.